shared/src/business/utilities/getFormattedCaseDetail.js

```diff
@@ -319,9 +319,15 @@ const formatCase = (applicationContext, caseDetail) => {
     // establish an initial sort by ascending index
     result.formattedDocketEntries.sort(byIndexSortFunction);
 
-    result.pendingItemsDocketEntries = result.formattedDocketEntries.filter(
-      entry => entry.pending && entry.servedAt,
-    );
+    if (isCodeEnabled(7198)) {
+      result.pendingItemsDocketEntries = result.formattedDocketEntries.filter(
+        entry => entry.pending && (entry.servedAt || entry.isLegacyServed),
```

Comment:
Here's another spot where it's checking for pending && (servedAt || isLegacyServed), so you could reuse that logic from the helper you make.

Reply:
Refactoring card added https://trello.com/c/BQLzXAuD